modules/flowable-ui/flowable-ui-idm-logic/src/main/java/org/flowable/ui/idm/service/keycloak/KeycloakConfiguration.java

```diff
@@ -204,12 +232,29 @@ public AccessToken fetchAccessToken() {
             }
         }
 
+        private MultiValueMap<String, String> createAccessTokenRequestParameters() {
+            MultiValueMap<String, String> tokenRequestBody = new LinkedMultiValueMap<>();
+            tokenRequestBody.add("grant_type", getGrantType());
+            tokenRequestBody.add("client_id", getClientId());
+            if (isClientCredentials()){
```

Comment:
small nitpick: add space before the brace `{`

Reply:
Done